pkg/chain/chain.go

```diff
@@ -112,4 +119,14 @@ type BondedECDSAKeep interface {
 
 	// LatestDigest returns the latest digest requested to be signed.
 	LatestDigest(keepAddress common.Address) ([32]byte, error)
+
+	// GetPublicKey returns keep's public key.
```

Comment:
What is returned when there is no public key yet? Worth being clear about it here.